modules/RTC/JitsiTrack.js

```diff
@@ -190,6 +190,36 @@ JitsiTrack.prototype.attach = function (container) {
     return container;
 };
 
+JitsiTrack.prototype.changeAudioOutput = function (audioOutputDeviceId) {
+    return Promise.all(this.containers.map(function(element) {
+        if (typeof element.setSinkId !== 'undefined') {
+            try {
+                return element.setSinkId(audioOutputDeviceId)
+                    .then(function () {
+                        console.log('Audio output device changed on element ' + element);
```

Comment:
@paweldomas reminded me recently that we should try using the internal logger and not the console log in the library, so I'm just echoing his advice here.